### PR TITLE
Reformat seedData.py

### PR DESCRIPTION
**Pull request by gussmith23**
### What issue is this solving?
This is an initial step towards fixing [#250](https://github.com/codeforpdx/dwellingly-app/issues/250).
I found that, while working towards fixing the linked issue, the current structure of the file was hard to read, mostly due to formatting and the use of integer literals for ID fields. This PR reformats the code and adds variable names. It also identifies two bugs to be fixed in the PR for the linked issue.

### Any helpful knowledge/context for the reviewer?
Shouldn't require re-seeding; this code should be equivalent to the previous code (according to the tests)!

### Please make sure you've attempted to meet the following coding standards
- [X] Code builds successfully
- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: data/seedData.py [hunk 1]
```diff
@@ -12,84 +12,236 @@
...
+    user_3.save_to_db()
+    user_mister_sir = UserModel(email="MisterSir@dwellingly.org",
+                                role=RoleEnum.PROPERTY_MANAGER,
+                                firstName="Mr.",
+                                lastName="Sir",
+                                password="1234",
+                                phone="555-555-5555",
+                                archived=0)
+    user_mister_sir.save_to_db()
+    user_gray_pouponn = UserModel(email="user3@dwellingly.org",
```
Comment (Chris-Boe) [inline data/seedData.py]: Looks like this user has the same email as `user3`, want to fix that while here?

File: data/seedData.py [hunk 2]
```diff
@@ -12,84 +12,236 @@
...
 
-    newTicket = TicketModel(issue="The roof, the roof, the roof is one fire.", tenant=1, sender=1, status="In Progress", urgency="Low", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Flaming Dumpster Fire.", tenant=2, sender=3, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Unpaid Rent", tenant=1, sender=1, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
-    newTicket = TicketModel(issue="Over 40 cats in domicile.", tenant=2, sender=3, status="Critical", urgency="HIGH", assignedUser=4)
-    db.session.add(newTicket)
+    # TODO(@gussmith23) This note doesn't actually attach to a real issue!
```
Comment (Chris-Boe) [inline data/seedData.py]: Were these notes to remind us to fix now or later?

**Review by Chris-Boe — DISMISSED** [review]
Lots better, thanks for cleaning this up! Just got some merge conflicts and a couple Q's

**Comment by Chris-Boe** [comment]
Also just a note about contributing, please try to clone the repo instead of forking, so we can pull it down to test easier! Thanks

File: data/seedData.py (hunk 1 above)
Comment (NickSchimek) [reply data/seedData.py]: good catch @Chris-Boe When this happens it should cause an error to propagate within the app. I created issue codeforpdx/dwellingly-app/issues/333 to address this.

File: data/seedData.py (hunk 2 above)
Comment (NickSchimek) [reply data/seedData.py]: @gussmith23 If this is an issue and needs to be fixed could you create an new issue instead of adding a TODO comment?

**Comment by NickSchimek** [comment]
This closes https://github.com/codeforpdx/dwellingly-app/issues/250

File: data/seedData.py (hunk 2 above)
Comment (gussmith23) [reply data/seedData.py]: It was to remind me to fix later; I will add an issue!

File: data/seedData.py (hunk 1 above)
Comment (gussmith23) [reply data/seedData.py]: Fixed!

File: data/seedData.py (hunk 2 above)
Comment (gussmith23) [reply data/seedData.py]: Added these as their own issues!

**Comment by gussmith23** [comment]
I haven't actually added any new test data as codeforpdx/dwellingly-app#250 requests. I've hopefully made it easier to add test cases though. I can handle adding some edge cases, though I don't quite have the familiarity yet to know what cases are actually edge cases :)

**Comment by gussmith23** [comment]
Merge conflicts resolved, please take a look!

**Review by NickSchimek — APPROVED** [review]
Thanks!
